controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Clusters.java

```diff
@@ -128,6 +128,19 @@ public boolean initialize(@PathParam("clusterName") String clusterName) {
   }
 
   /**
+   * Remove a given cluster. This may include removing designated tag
+   * in DB and/or writing shard config to zookeeper.
```

Comment:
In this case, I think we can assume the cluster tag should always exist in DB. So you can just word it like "remove tag in DB and shard config in zookeeper".